Gemfile

```diff
@@ -149,6 +149,7 @@ group :development do
   group :test do
     gem 'coveralls', '~> 0.7.4', require: false
     gem 'capybara', '~> 2.13.0'
+    gem 'capybara-screenshot'
```

Comment:
It automatically saves screenshots in html and png for each feature spec failure.  You'll be free of having to manually do that and rerun the failed specs.